source/userguide/getting_started.rst

```diff
@@ -24,138 +24,78 @@ If you're going to be using MindMeld often, **we recommend you do the virtualenv
 Install with Docker
 ^^^^^^^^^^^^^^^^^^^
 
-The ``Dockerfile`` provided by MindMeld contains MindMeld and all its dependencies. Follow these steps to get started using Docker:
+In the Docker setup, we use Docker only for running MindMeld dependencies, namely, Elasticsearch and the Numerical Parser service. The developer will then use their local machine to run the MindMeld project.
+
+1. Pull and run Docker container
+""""""""""""""""""""""""""""""""
 
 #. First, `install Docker <https://www.docker.com/community-edition#/download>`_, and run it.
-#. Then, open a terminal (shell) and run this command:
+#. Then, open a terminal (shell) and run these commands:
 
 .. code-block:: shell
 
-   docker pull mindmeldworkbench/mindmeld
-   docker run -p 0.0.0.0:7150:7150 mindmeldworkbench/mindmeld -ti -d
-
-The Docker container contains Elasticsearch, the numerical parsing service, the MindMeld library and the Home Assistant application for you to test. The container will build and serve the application on port 7150 which is exposed to the external environment.
-
-The application code and data is located at directory ``/root/home_assistant`` on the docker container.
+   docker pull mindmeldworkbench/dep:latest
+   docker run -p 0.0.0.0:9200:9200 -p 0.0.0.0:7151:7151 -p 0.0.0.0:9300:9300 mindmeldworkbench/dep -ti -d
 
```

Comment:
Can we include a specific note in the documentation that ports 7151, 9200 and 9300 will be used by the Docker container, so users should make sure that any other applications don't clash with those ports. And if they already have ES installed and are using it for other applications, then virtualenv setup is recommended